test/project/uiengine.config.js

```diff
@@ -51,6 +52,24 @@ module.exports = {
       module: '@uiengine/adapter-webpack',
       options: webpackAdapterReactOptions
     },
+    njk: {
+      module: '/Users/cos/Playground/uiengine/packages/adapter-nunjucks',
```

Comment:
I think this is why the tests fail :)

Reply:
I've updated the path to a relative path but the tests are still failing. Is there a specific pattern I need to follow to reference the module in here?

Reply:
I think I fixed it with ebfb45f. There was also [this one](https://github.com/dennisreimann/uiengine/pull/85/commits/ebfb45ff3e195fc640fc1cf6f6e0871e9b1f95d9#diff-4b364b41b339d2f6ecd774d156ab1feea93189b6fdf33465f21a639854100e0e).